### PR TITLE
Fix some PHP codes

### PR DESCRIPTION
**Pull request by atailouloute**

### CODE REVIEW

File: templating/namespaced_paths.rst [hunk 1]
```diff
@@ -68,7 +68,7 @@ directory:
         $container->loadFromExtension('twig', array(
             'paths' => array(
                 '%kernel.root_dir%/../vendor/acme/foo-bar/templates' => 'foo_bar',
-            );
+            )
```
Comment (xabbuh) [inline templating/namespaced_paths.rst]: we should add a trailing comma here

**Comment by xabbuh** [comment]
Thank you for the many good catches and improvements @atailouloute! To ease the merging of all the small fixes I will combine your PRs in a single PR (see #7737) and merge that one instead.

File: assetic/jpeg_optimize.rst [hunk 2]
```diff
@@ -294,7 +293,7 @@ file:
             'twig' => array(
                 'functions' => array(
                     'jpegoptim' => array(
-                        output => 'images/*.jpg'
+                        'output' => 'images/*.jpg'
```
Comment (HeahDude) [inline assetic/jpeg_optimize.rst]: There should be a trailing comma here too.

Comment (atailouloute) [inline assetic/jpeg_optimize.rst]: The PR is closed, I'll do this changes in another commit

Comment (atailouloute) [inline assetic/jpeg_optimize.rst]: Fixed in #7741

**Pull request closed without merging**